### PR TITLE
Merge commits from master into develop

### PR DESCRIPTION
**Pull request by cbeer**
Part of #166.

### CODE REVIEW

File: lib/dor/datastreams/workflow_ds.rb [hunk 1]
```diff
@@ -24,7 +23,7 @@ def get_workflow(wf, repo = 'dor')
       Workflow::Document.new(xml.to_s)
     end
 
-    alias_method :[], :get_workflow
+    alias :[] :get_workflow
```
Comment (darrenleeweber) [inline lib/dor/datastreams/workflow_ds.rb]: I'd like to understand this better.  I've read recently that one or the other is more problematic.  From my recent reading, I concluded that `alias_method` is the way to go, but I'm not entirely clear on why one or the other is better.  For example, http://stackoverflow.com/questions/4763121/should-i-use-alias-or-alias-method

File: .rubocop_todo.yml [hunk 2]
```diff
@@ -117,11 +117,6 @@ Metrics/ClassLength:
 Metrics/CyclomaticComplexity:
   Max: 24
 
-# Offense count: 60
-# Configuration parameters: AllowURI, URISchemes.
-Metrics/LineLength:
-  Max: 6121
-
```
Comment (darrenleeweber) [inline .rubocop_todo.yml]: By removing this and running rubocop again, does it complain about long lines?  Have they been fixed?

File: lib/dor/datastreams/workflow_ds.rb [hunk 3]
```diff
@@ -9,7 +8,7 @@ class WorkflowDs < ActiveFedora::OmDatastream
         t.process {
           t.name_(    :path => {:attribute => 'name'     }, :index_as => [:displayable, :not_searchable] )
           t.status(   :path => {:attribute => 'status'   }, :index_as => [:displayable, :not_searchable] )
-          t.timestamp(:path => {:attribute => 'datetime' }, :index_as => [:displayable, :not_searchable] ) # , :data_type => :date)
+          t.timestamp(:path => {:attribute => 'datetime' }, :index_as => [:displayable, :not_searchable] ) #, :data_type => :date)
```
Comment (darrenleeweber) [inline lib/dor/datastreams/workflow_ds.rb]: Has a library change (ActiveFedora?) made the extra `:data_type` arg obsolete and/or unnecessary?  Why leave the comment?

File: lib/dor/datastreams/workflow_ds.rb (hunk 1 above)
Comment (cbeer) [inline lib/dor/datastreams/workflow_ds.rb]: I believe @atz used `alias`, as suggested by the ruby style guide + rubocop:

https://github.com/bbatsov/ruby-style-guide#alias-method-lexically

Comment (cbeer) [inline lib/dor/datastreams/workflow_ds.rb]: That said, I wouldn't worry too much about it. I think we should try to remove the `[]` accessor entirely. It seems to be used in one test in `dor-services`, but i'll take a look in some other projects just to be sure.

File: .rubocop_todo.yml (hunk 2 above)
Comment (cbeer) [inline .rubocop_todo.yml]: Yes, it complains about long lines. And indeed, in a future commit, someone should fix that. I'd really rather not muddle getting `master` and `develop` back in sync with additional changes.

I'd also be concerned more if `dor-services` had a clean bill of health from rubocop, but there are already 200 violations (and the vast majority are not line length complaints).

File: lib/dor/datastreams/workflow_ds.rb (hunk 3 above)
Comment (cbeer) [inline lib/dor/datastreams/workflow_ds.rb]: Who knows. From the history, @atz was merely addressing rubocop violations in this file, and not necessarily trying to understand previous practice.

**Comment by darrenleeweber** [comment]
I'm concerned that this merge is backwards and should be `develop` into `master`.

**Comment by cbeer** [comment]
It should, except `master` got ahead of `develop` by these commits. This is just trying to set our development and release practices right again by getting the branches back in sync.

**Pull request merged**